partiql-cli/src/main/kotlin/org/partiql/cli/shell/Shell.kt

```diff
@@ -185,35 +191,34 @@ internal class Shell(
             val command = when (val end: Int = CharMatcher.`is`(';').or(CharMatcher.whitespace()).indexIn(line)) {
                 -1 -> ""
                 else -> line.substring(0, end)
-            }
-            when (command.toLowerCase(Locale.ENGLISH).trim()) {
+            }.toLowerCase(Locale.ENGLISH).trim()
+            when (command) {
                 "!exit" -> return
                 "!add_to_global_env" -> {
                     // Consider PicoCLI + Jline, but it doesn't easily place nice with commands + raw SQL
                     // https://github.com/partiql/partiql-lang-kotlin/issues/63
```

Comment:
Not sure what this comment applied to or whether it is still useful, but the code that it was in front of is now inside requireInput().